_plugins/jekyll_rummageable/indexer.rb

```diff
@@ -34,12 +36,22 @@ def generate(site)
         page_paragraphs = extract_text(site, item)
 
         {
-          "title"             => item.data['title'] || item.name ,
+          "_type"             => "manual_section",
+          "organisations"     => ["government-digital-service"],
+          "manual"            => "service-manual",
+          "title"             => "#{MANUAL_TITLE}: #{item.data['title'] || item.name}",
           "indexable_content" => page_paragraphs.join(" ").gsub("\r"," ").gsub("\n"," "),
           "description"       => extract_summary(site, item),
           "link"              => item.url
         }
       end
+      index << {
+        "_type"             => "manual",
+        "organisations"     => ["government-digital-service"],
+        "title"             => MANUAL_TITLE,
+        "description"       => "From April 2014, digital services from the government must meet the new Digital by Default Service Standard",
```

Comment:
This text has [changed now](https://github.com/alphagov/government-service-design-manual/commit/edf15e5073ede79551ed970ee7e962d29777aea7).